Makefile

```diff
@@ -669,6 +672,14 @@ test-capd-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests
 kind-cluster: ## Create a new kind cluster designed for development with Tilt
 	hack/kind-install-for-capd.sh
 
+.PHONY: clean-kind
```

Comment:
Can we move this below the clean-charts targets (L966-L968) so it will appear in the Cleanup / Verification section of the make help?